packages/client/hmi-client/src/components/code/tera-code-dynamic.vue

```diff
@@ -134,9 +134,8 @@ ul {
 	display: flex;
 }
 
-.edit-input-container:deep(.p-inputnumber-input) {
+.edit-input-container .tera-input {
 	width: 107px;
```

Comment:
could you use `class=w-tailwindnumber` instead

Reply:
I had to do something a little different here to do use the full width, `tera-input :deep(input)`